lib/diplomat/kv.rb

```diff
@@ -119,23 +119,23 @@ def delete key, options=nil
     private
 
     def check_acl_token
-      use_named_parameter("token", Diplomat.configuration.acl_token)
+      use_named_parameter('token', Diplomat.configuration.acl_token)
     end
 
     def use_cas(options)
-      if options then use_named_parameter("cas", options[:cas]) else [] end
+      options ? use_named_parameter('cas', options[:cas]) : []
     end
 
     def use_consistency(options)
-      if options && options[:consistency] then ["#{options[:consistency]}"] else [] end
+      options && options[:consistency] ? ["#{options[:consistency]}"] : []
     end
 
     def recurse_get(options)
-      if options && options[:recurse] then ['recurse'] else [] end
+      options && options[:recurse] ? ['recurse'] : []
     end
 
     def dc(options)
-      if options && options[:dc] then use_named_parameter("dc", options[:dc]) else [] end
+      options && options[:dc] ? use_named_parameter('dc', options[:dc]) : []
```

Comment:
This isn't completely obvious how it parses. Does it parse as:
     ( options && options[:dc] ) ? use_named_parameter('dc', options[:dc]) : []
or as:
      options && (options[:dc] ? use_named_parameter('dc', options[:dc]) : [])
The parenthesis might be redundant, but it will help people who are reading this in the future from scratching their heads and wondering about it.